tests/rules/test_fix_file.py

```diff
@@ -14,47 +14,47 @@
 a.c:3:1: error: expected expression before '}' token
  }
   ^
-"""),
+"""),  # noqa
 
 ('clang a.c', 'a.c', 3, 1, '',
 """
 a.c:3:1: error: expected expression
 }
 ^
-"""),
+"""),  # noqa
```

Comment:
Oh, so it does! I think that's the best option so far, and I've updated the branch accordingly :D